pht_federated/aggregator/services/discovery/column_harmonization.py

```diff
@@ -0,0 +1,217 @@
+from typing import List, Tuple
+
+from fuzzywuzzy import fuzz
+
+from pht_federated.aggregator.schemas.dataset_statistics import DatasetStatistics
+
+
+def compare_two_datasets(
+    dataset_statistics: Tuple[DatasetStatistics, str],
+    aggregator_statistics: DatasetStatistics,
+) -> dict:
+    """
+    Compares two datasets in form of DatasetStatistics objects with respect to their column names
+    :param dataset_statistics: DatasetStatistics of local dataset
+    :param aggregator_statistics: DatasetStatistics of aggregator dataset
+    :return: Dictionary which lists the differences between the two datasets
+    """
+
+    dataset_name = dataset_statistics[1]
+    df_stats_dict = dataset_statistics[0].dict()
+    aggregator_stats_dict = aggregator_statistics.dict()
+
+    df_stats_dict = df_stats_dict["column_information"]
+    aggregator_stats_dict = aggregator_stats_dict["column_information"]
```

Comment:
simplify into single assignment operation and find better variable names i.e. input_column_information, aggregator_column_information